nucypher/cli/painting.py

```diff
@@ -385,3 +393,77 @@ def paint_deployment_delay(emitter, delay: int = 3) -> None:
     for i in range(delay)[::-1]:
         emitter.echo(f"{i}...", color='yellow')
         time.sleep(1)
+
+
+def paint_stakers(emitter, stakers: List[str], agent) -> None:
+    current_period = agent.get_current_period()
+    emitter.echo(f"\nCurrent period: {current_period}")
+    emitter.echo("\n| Stakers |\n")
+    emitter.echo(f"{'Checksum address':42}  Staker information")
+    emitter.echo('=' * (42 + 2 + 53))
+
+    for staker in stakers:
+        nickname, pairs = nickname_from_seed(staker)
+        symbols = f"{pairs[0][1]}  {pairs[1][1]}"
+        emitter.echo(f"{staker}  {'Nickname:':10} {nickname} {symbols}")
+        tab = " " * len(staker)
+        info = agent.get_staker_info(staker)
+        stake, confirmed1, confirmed2, restake, *remaining_info = info
+        locked_restake_until_period, worker, worker_start, last_active = remaining_info
+
+        last_confirmed_period = max(confirmed1, confirmed2)
```

Comment:
It's not correct for all cases, better to use `getLastActivePeriod()`, which handle logic for idle workers

Reply:
✅